rubygem-font-awesome-sass/rubygem-font-awesome-sass.spec

```diff
@@ -0,0 +1,68 @@
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%global gem_name font-awesome-sass
+
+Name: %{?scl_prefix}rubygem-%{gem_name}
+Version: 4.3.2.1
+Release: 1%{?dist}
+Summary: Font-Awesome SASS
+Group: Development/Languages
+License: MIT
+URL: https://github.com/FortAwesome/font-awesome-sass
+Source0: http://rubygems.org/gems/%{gem_name}-%{version}.gem
+Requires: %{?scl_prefix_ruby}ruby(release)
+Requires: %{?scl_prefix_ruby}ruby(rubygems)
+Requires: %{?scl_prefix_ruby}ruby
+Requires: %{?scl_prefix_ror}rubygem(sass) >= 3.2
```

Comment:
https://rubygems.org/gems/font-awesome-sass & https://github.com/FortAwesome/font-awesome-sass/blob/master/font-awesome-sass.gemspec say it's `>= 3.2`, where did you see the `~>`?